Client/Frontend/Browser/PageZoom/PageZoomView.swift

```diff
@@ -23,70 +23,56 @@ private struct ZoomView: View {
   
   var body: some View {
     HStack(spacing: 0.0) {
-      Button(action: {
-        onDecrement()
-      }) {
+      Button(action: onDecrement) {
         Image(systemName: "minus")
           .font(.system(.footnote).weight(.medium))
-          .foregroundColor(color)
-          .frame(width: 44.0)
-          .padding(.vertical, 8.0)
-          .contentShape(Rectangle())
+          .foregroundColor(Color(UIColor.braveLabel))
+          .frame(width: buttonWidth)
       }
-      .fixedSize()
       .disabled(value == minValue)
       
       Divider()
-        .padding(.vertical, 8.0)
       
-      Text(PageZoomView.percentFormatter.string(from: .init(value: 0.999)) ?? "999%")
+      Text(NSNumber(value: 0.999), formatter: PageZoomView.percentFormatter)
         .font(.system(.footnote).weight(.medium))
-        .fixedSize()
+        .fixedSize(horizontal: true, vertical: false)
         .hidden()
         .overlay(resetZoomButton)
         .padding(.horizontal)
-        .padding(.vertical, 8.0)
       
       Divider()
-        .padding(.vertical, 8.0)
       
-      Button(action: {
-        onIncrement()
-      }) {
+      Button(action: onIncrement) {
         Image(systemName: "plus")
           .font(.system(.footnote).weight(.medium))
-          .foregroundColor(color)
-          .frame(width: 44.0)
-          .padding(.vertical, 8.0)
-          .contentShape(Rectangle())
+          .foregroundColor(Color(UIColor.braveLabel))
+          .frame(width: buttonWidth)
       }
-      .fixedSize()
       .disabled(value == maxValue)
     }
+    .padding(.vertical, 8.0)
     .fixedSize(horizontal: false, vertical: true)
     .background(Color(UIColor.secondaryBraveBackground))
     .clipShape(RoundedRectangle(cornerRadius: 10.0, style: .continuous))
   }
   
   private var resetZoomButton: some View {
-    Button(action: {
-      onReset()
-    }) {
-      Text(PageZoomView.percentFormatter.string(from: .init(value: value)) ?? "\(Int(value * 100.0))%")
+    Button(action: onReset) {
+      Text(NSNumber(value: value), formatter: PageZoomView.percentFormatter)
         .font(.system(.footnote).weight(.medium))
-        .foregroundColor(color)
+        .foregroundColor(Color(UIColor.braveLabel))
         .padding()
         .contentShape(Rectangle())
     }
-    .fixedSize()
+    .fixedSize(horizontal: true, vertical: false)
```

Comment:
Don't need this fixed size (the layout is determined by the parent already, this will always be overlayed on top)

Reply:
Doesn't render at all if I remove it. `overlay` doesn't seem to inherit the parent's fixedSize.